openmedia/gui/controlbox.py

```diff
@@ -74,14 +85,35 @@ def _play_next(self, widget):
         if self.playlist_box.is_visible():
             self.playlist_box.show_all()
 
+    def _play_previous(self, widget):
+        Player.instance().play_previous()
+        if self.playlist_box.is_visible():
```

Comment:
Yep, it seems to be unnecessary. I'll remove it from `_play_previous` and `_play_next`.